logging-dropwizard-3/pom.xml

```diff
@@ -7,48 +7,44 @@
     <artifactId>logging-dropwizard-3</artifactId>
     <packaging>jar</packaging>
 
-    <properties>
-        <dropwizard.version>3.0.3</dropwizard.version>
-        <hamcrest.version>2.2</hamcrest.version>
-    </properties>
-
     <parent>
         <artifactId>pay-java-commons</artifactId>
         <groupId>uk.gov.service.payments</groupId>
         <version>1.0.0</version>
     </parent>
     <dependencies>
+        <!-- Main dependencies that are imported from one of the BOMs specified
+             in <dependencyManagement> in the parent pom, so no explicit versions needed -->
         <dependency>
-            <groupId>org.dhatim</groupId>
-            <artifactId>dropwizard-sentry</artifactId>
-            <version>2.1.6</version>
+            <groupId>io.dropwizard</groupId>
+            <artifactId>dropwizard-core</artifactId>
         </dependency>
         <dependency>
             <groupId>io.dropwizard</groupId>
             <artifactId>dropwizard-json-logging</artifactId>
-            <version>${dropwizard.version}</version>
+        </dependency>
+
+        <!-- Main dependencies that need explicit versions -->
+        <dependency>
+            <groupId>org.dhatim</groupId>
+            <artifactId>dropwizard-sentry</artifactId>
+            <version>2.1.6</version>
         </dependency>
         <dependency>
             <groupId>net.logstash.logback</groupId>
             <artifactId>logstash-logback-encoder</artifactId>
             <version>7.4</version>
         </dependency>
-        <dependency>
-            <groupId>io.dropwizard</groupId>
-            <artifactId>dropwizard-core</artifactId>
-            <version>${dropwizard.version}</version>
-        </dependency>
         <dependency>
             <groupId>com.google.code.gson</groupId>
             <artifactId>gson</artifactId>
             <version>2.11.0</version>
         </dependency>
-        <dependency>
-            <groupId>org.glassfish.jersey.bundles.repackaged</groupId>
-            <artifactId>jersey-guava</artifactId>
-            <version>2.25.1</version>
-        </dependency>
```

Comment:
I assume this is no longer needed?

Reply:
The only place it was used was https://github.com/alphagov/pay-java-commons/pull/864/files#diff-b51799d1bc690ddea69f86f4c3de8790ff334641f28fd34da7d814ec8eae73abL4